Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs

```diff
@@ -97,64 +97,61 @@ private void InitializeInternal()
             bool addedComponents = false;
 
 #if UNITY_EDITOR
-
-            var eventSystems = UnityEngine.Object.FindObjectsOfType<EventSystem>();
-            var standaloneInputModules = UnityEngine.Object.FindObjectsOfType<StandaloneInputModule>();
-
-            CameraCache.Main.transform.position = Vector3.zero;
-            CameraCache.Main.transform.rotation = Quaternion.identity;
-
-            if (eventSystems.Length == 0)
+            if (!UnityEditor.EditorApplication.isPlaying)
             {
-                FocusProvider.UIRaycastCamera.gameObject.EnsureComponent<EventSystem>();
-                addedComponents = true;
-            }
-            else
-            {
-                bool raiseWarning;
+                var eventSystems = UnityEngine.Object.FindObjectsOfType<EventSystem>();
+                var standaloneInputModules = UnityEngine.Object.FindObjectsOfType<StandaloneInputModule>();
 
-                if (eventSystems.Length == 1)
+                CameraCache.Main.transform.position = Vector3.zero;
+                CameraCache.Main.transform.rotation = Quaternion.identity;
+
+                if (eventSystems.Length == 0)
                 {
-                    raiseWarning = eventSystems[0].gameObject != focusProvider.gameObject;
+                    FocusProvider.UIRaycastCamera.gameObject.EnsureComponent<EventSystem>();
```

Comment:
Do we have guidelines on using the public vs the private inside the same class?
I feel like I've seen @SimonDarksideJ comment about using the private.

Reply:
How does that apply here?

Reply:
`FocusProvider` vs `focusProvider`.

Your previous commit changed which one was being used.

![image](https://user-images.githubusercontent.com/3580640/43485875-4a2c6c7c-94c7-11e8-9f17-a45d31ca8890.png)

Reply:
Depends on if the property does anything extra when accessing it. I can revert if you'd like.